.github/workflows/devpackpublish.yml

```diff
@@ -48,23 +48,4 @@ jobs:
           git config --local user.email "action@github.com"
           git add .
           git commit -m "[CI Skip] ci: publish prerelease" -n
-          git push
-  dispatch:
```

Comment:
Is it ok to remove these outdated triggers?

Reply:
yes